test/views/declaration/spec/UnitViewSpec.scala

```diff
@@ -41,8 +44,7 @@ class UnitViewSpec extends UnitSpec with ViewMatchers {
   }
 }
 
-object UnitViewSpec {
-  val realMessagesApi: MessagesApi = {
-    new GuiceApplicationBuilder().injector().instanceOf(classOf[MessagesApi])
-  }
+object UnitViewSpec extends Injector {
+  val realMessagesApi: MessagesApi = instanceOf[MessagesApi]
+  val realAppConfig: AppConfig = instanceOf[AppConfig]
```

Comment:
I'm not sure that putting `realAppConfig` in UnitViewSpec is good idea. In practice it's not used in every spec but it'll be injected.

Reply:
Changed to use mock. You're probably right.